res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -381,6 +416,21 @@ PioneerDDJSX.init = function(id) {
             PioneerDDJSX.initDeck(index);
         }
     }
+
+    // init effects section:
+    PioneerDDJSX.effectUnit = [];
+    PioneerDDJSX.effectUnit[1] = new PioneerDDJSX.EffectUnit(1);
```

Comment:
Change `PioneerDDJSX.EffectUnit` to `components.EffectUnit`. Also use `[1, 3]` and `[2, 4]` to allow shift + Tap to toggle between the effect units being controlled.